### PR TITLE
Remove dispatch interceptors in C#

### PR DESCRIPTION
**Pull request by bernardnormier**
This PR removes dispatch interceptors in C#. It's the first step in the server-side dispatch refactoring.

### CODE REVIEW

File: csharp/test/Ice/namespacemd/msbuild/client/client.csproj [hunk 1]
```diff
@@ -8,6 +8,7 @@
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <OutputPath>.</OutputPath>
+    <Configurations>Release;Debug</Configurations>
```
Comment (pepone) [inline csharp/test/Ice/namespacemd/msbuild/client/client.csproj:11]: Why is this required?

**Review by pepone — APPROVED** [review]

File: csharp/test/Ice/namespacemd/msbuild/client/client.csproj (hunk 1 above)
Comment (bernardnormier) [reply csharp/test/Ice/namespacemd/msbuild/client/client.csproj:11]: This was added by VS Studio when I removed the interceptor test.
